webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricsManager.js

```diff
@@ -19,6 +20,15 @@ export class SakaiRubricsManager extends RubricsElement {
     this.siteRubricsExpanded = "true";
     this.sharedRubricsExpanded = "false";
     this.enablePdfExport = false;
+    this.isSuperUser = false;
+  }
+
+  attributeChangedCallback(name, oldValue, newValue) {
+    super.attributeChangedCallback(name, oldValue, newValue);
+
+    if (name === "is-super-user") {
+      this.isSuperUser = newValue === "true";
```

Comment:
Why did you need to do this? This seems a bit strange. You should just be able to let lit's setter sort this out from from the attribute.

Reply:
Yes, you're right. The reason I did it this way was to handle converting the is-super-user attribute (which is always a string in HTML) into a boolean (true or false) value for the property. But I realize that Lit can handle this automatically if we use its built-in property reflection. So I'm going to modify it. Thanks a lot for pointing that out.

Reply:
Hi Adrian, I’ve been reviewing the implementation, and after considering the collaboration and recommendations from LIT contributors, I believe it’s best to keep the attributeChangedCallback method as it is.

The main reason for this is that in LIT, boolean attributes are interpreted solely based on their presence in the DOM. This means that even if the is-super-user attribute is passed as false, the component will interpret it as true simply because the attribute exists. To avoid confusion and ensure the component’s logic is accurate, it’s necessary to convert it to a real boolean value.

By using the attributeChangedCallback method, we can ensure that isSuperUser is stored correctly as a real boolean with this.isSuperUser = newValue === "true";.

Reply:
It's a shame we can't make Thymeleaf conditionally add that boolean attribute. Boolean attributes are not supposed to be set to a value because that just doesn't make sense. However, I see the issue. Thymeleaf just won't let you conditionally add an attribute unless it is one of the standard dialect th:checked type attributes.

Reply:
However ...

https://stackoverflow.com/questions/64803308/thymeleaf-add-attribute-without-value

Reply:
That way works perfectly, and there's no need to use the attributeChangedCallback method. Thank you for sharing the StackOverflow link!

I encountered the same issue with enablePdfExport, and I resolved it in the same way as with isSuperUser. I agree that the problem lies with Thymeleaf not allowing custom boolean attributes, so putting the hacky code in the Thymeleaf template makes sense. Thanks again!